scripts/update-repo-internal.sh

```diff
@@ -16,17 +16,20 @@ RELEASE="${DISTRO##*-}"
 case $PARENT in
     centos)
         mkdir -p repo/$PARENT/$RELEASE/packages
-        find package/$DISTRO/ -name *.rpm | \
+        find package/$DISTRO/ -name i'*.rpm' | grep -v lstore-release.rpm | \
```

Comment:
Well, we need to make the lio_* commands have sensible command line parsing before we start documenting all of the things. Right now, "lio_getattr -h" does nothing...